libraries/wasm/examples/node.js

```diff
@@ -0,0 +1,160 @@
+const identity = require('../node/iota_identity_wasm')
+
+console.log("identity", identity)
+
+// ======================================================
+// https://github.com/seanmonstar/reqwest/issues/910
+// ======================================================
+const fetch = require('node-fetch')
+global.Headers = fetch.Headers
+global.Request = fetch.Request
+global.Response = fetch.Response
+global.fetch = fetch
+
+const { initialize, resolve, publish, checkCredential, Key, PubKey, Doc, DID, VerifiableCredential } = identity
+
+initialize()
+
+async function playground() {
+  console.log("key", Key.ed25519())
+
+  console.log("did", new DID(Key.ed25519()))
+
+  console.log("did", new DID(Key.ed25519(), "com"))
+
+  const { key, doc } = Doc.generateEd25519("com")
+
+  console.log("key (generated)", key)
+  console.log("doc (generated)", doc)
+
+  console.log("doc (unsigned)", doc.toJSON())
+
+  doc.sign(key)
+
+  console.log("doc (signed)", doc.toJSON())
+
+  console.log("doc valid?", doc.verify())
+
+  const json = JSON.stringify(doc.toJSON())
+
+  console.log("From JSON >", Doc.fromJSON(json))
```

Comment:
fromJSON is not actually loading from a JSON object, but from a string that contains a valid JSON. Causes confusion as I did JSON.parse() before calling the function. Either rename or rework the logic to be more consistent with expectations.